app/components/logger.js

```diff
@@ -10,7 +10,7 @@ module.exports = function(app, options) {
     const logger = {
         passToConsole(operation, args) {
             if (logEnabled) {
-                console[operation].apply(console, args);
+                console[operation].call(console, ...args);
```

Comment:
May I misunderstood something here but why do we need to invoke the console function call instead of just calling the function?
Whats the benefit of invoking a function with call instead of apply? ( I know there is a different interface for the arguments)